addons/l10n_latam_check/wizards/account_payment_register.py

```diff
@@ -42,3 +50,11 @@ def _create_payment_vals_from_wizard(self, batch_result):
                 'l10n_latam_move_check_ids': [Command.link(x.id) for x in self.l10n_latam_move_check_ids]
             })
         return vals
+
+    def action_create_payments(self):
+        if self._is_latam_check_payment(check_subtype="move_check") and (
+            len(self.l10n_latam_move_check_ids.mapped("currency_id")) > 1
+            or self.l10n_latam_move_check_ids[:1].currency_id != self.currency_id
+        ):
+            raise ValidationError(_("You can't mix checks with different currencies. Please create separate payments for checks of each currency."))
```

Comment:
@kitan191 @jorenvo what about something like:
```suggestion
            raise ValidationError(
    _(
        "You can't mix checks of different currencies in one payment, "
        "and you can't change the payment's currency if checks are already created in that currency. "
        "Please create separate payments for each currency."
    )
)
```

Reply:
Looks good to me. :ok_hand: 
What would be the Spanish (Latin America) translation for it? :slightly_smiling_face:

Reply:
```
        "No se pueden mezclar cheques de distintas monedas en un mismo pago, "
        "y no se puede cambiar la moneda del pago si los cheques seleccionados están creados en esa moneda. "
        "Por favor, crea pagos separados para cada moneda."
```
@kitan191

Reply:
I've updated the fix.

Reply:
@kitan191, thanks for adding the changes.
@jorenvo I've tested and it looks good from my side.

Thanks!